src/main/java/uk/gov/ons/ctp/response/casesvc/endpoint/CaseEndpoint.java

```diff
@@ -103,6 +104,29 @@ public CaseEndpoint(
     return ResponseEntity.ok(buildDetailedCaseDTO(caseObj, caseevents, iac));
   }
 
+  @RequestMapping(value = "/sampleunitids", method = RequestMethod.GET)
+  public ResponseEntity<CaseDetailsDTO[]> findCasesBySampleUnitIds(
+      @RequestParam(value = "sampleUnitId") final UUID[] sampleUnitIds,
```

Comment:
why this one is not directly a List ?

Reply:
This makes sense, then we don't need to do anything else to stream it